src/http/response.ts

```diff
@@ -44,22 +45,84 @@ export default class Response {
    */
   public status_code: number = Status.OK;
 
+  /**
+   * @description
+   *     A property to hold the path to the users views directory
+   *     from their project root
+   *
+   * @property string views_path
+   */
+  private views_path: string | undefined;
+
+  /**
+   * @description
+   *     The render method extracted from dejs
+   *
+   * @property any views_renderer
+   */
+  private template_engine: boolean | undefined = false;
+
   // FILE MARKER: CONSTRUCTOR //////////////////////////////////////////////////
 
   /**
    * @description
    *     Construct an object of this class.
    *
    * @param any request
+   *
+   * @param ResponseOptions options
+   *     See Drash.Interfaces.ResponseOptions
    */
-  constructor(request: any) {
+  constructor(request: any, options: ResponseOptions = {}) {
```

Comment:
instead of importing `ResponseOptions`, use `Drash.Http.ResponseOptions`

Reply:
Addressed this in the below commit `Improved the options parameter interface for Drash.Http.Response`